package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@mediamonks/eslint-config",
+  "version": "1.0.0",
+  "description": "MediaMonks Frontend Coding Standards Eslint Config",
+  "main": "index.js",
+  "scripts": {
+    "prettify": "prettier --write \"index.js\"",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mediamonks/frontend-coding-standards.git"
+  },
+  "keywords": [
+    "MediaMonks",
+    "Coding",
+    "Standards",
+    "eslint",
+    "eslintconfig",
+    "Frontend",
+    "JavaScript",
+    "TypeScript",
+    "React",
+    "Vue",
+    "Muban"
+  ],
+  "author": "MediaMonks",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/mediamonks/frontend-coding-standards/issues"
+  },
+  "homepage": "https://github.com/mediamonks/frontend-coding-standards#readme",
+  "peerDependencies": {
+    "@typescript-eslint/eslint-plugin": ">= 3",
```

Comment:
Maybe have a inclusive range for dependencies, so we know which versions this is tested on. Will require a bit more maintenance on this config.

Reply:
> You should declare your dependency on ESLint in package.json using the peerDependencies field. The recommended way to declare a dependency for future proof compatibility is with the ">=" range syntax, using the lowest required ESLint version

See https://eslint.org/docs/developer-guide/shareable-configs